libs/components/lists/src/lib/modules/repeater/repeater.component.spec.ts

```diff
@@ -102,6 +101,7 @@ describe('Repeater item component', () => {
   });
 
   it('should default expand mode to "none" when no expand mode is specified', fakeAsync(() => {
+    // revisit
```

Comment:
?

Reply:
I don't remember. Doesn't seem to be failing or anything, so I'm removing this.